commander.js

```diff
@@ -0,0 +1,23 @@
+const program = require('commander');
+program.version('0.0.1');
+
+program
+    .option('-no-commit', 'no commit step')
+    .option('--no-install', 'no install step')
+
+program.parse(process.argv);
+if (program.no-commit) {
+    if (program.no-install) {
+
+    } else {
+
+    }
+} else if (program.no-install) {
+    if (program.no-commit) {
+
+    } else {
+
+    }
+} else {
+
+}
```

Comment:
this should not be in a dedicated file (for now), we should include this logic directly in generator.js

after it is working, there can be a follow up PR (which I should make an issue for really) for breaking out and testing the code in isolation.